### ISSUE TITLE
error:140770FC:SSL routines:SSL23_GET_SERVER_HELLO:unknown protocol

### ISSUE THREAD

**Issue by nononymousse** [transmission-remote-gui/transgui#1445]
My setup :
- Client : Windows 11 22H2 w/ TransGUI v5.18.0 x64
- Server : Debian 11 w/ Transmission 3.00

Hi \^^

i did want to run TransGUI on my desktop but each time it try to connect to my server, i get this error : error:140770FC:SSL routines:SSL23_GET_SERVER_HELLO:unknown protocol

I did read some issues here and put OpenSSL files on the folder but nothing change

Don't know if it's related but i did enabled the web gui with TLS 1.3

Thank you for all your future answers, it will really help \^^

**Comment by xavery** [comment]
Transgui uses OpenSSL 1.0.2, which doesn't support TLS 1.3.

**Comment by xavery** [comment]
FYI, I switched [my fork](https://github.com/xavery/transgui) to OpenSSL v3 but you'd have to build transgui yourself as binaries are not yet available.

**Comment by Mrnofish** [comment]
> binaries are not **yet** available

So you do have plans? Watched & Starred!

**Comment by xavery** [comment]
> > binaries are not **yet** available
> 
> So you do have plans? Watched & Starred!

Yes, eventually I'd like to make builds straight in GitHub Actions but there's a lot of stuff to figure out, so I can't make any promises yet.

**Comment by nononymousse** [comment]
thx for your answers, i will look for that fork ☺️ 
(not very experienced with compiling in general and i never done it on Windows. Wish me luck lol xD)

(btw i wont close this thread unless the project dosen't use TLS 1.3, that fair imo but i could be wrong)

**Comment by xavery** [comment]
@nononymousse Windows builds of my fork are now available, please use the Actions menu to access the recent artifacts made from `master` like [this one](https://github.com/xavery/transgui/suites/11651232236/artifacts/605111161). If you encounter any issues, please report them in the fork.

**Comment by nononymousse** [comment]
Thanks a lot for your answer ^^

**Comment by PeterDaveHello** [comment]
Replied and closed by Codex:

> The final report in this thread confirms that the connection works after using the reverse proxy's HTTPS port, and the current master branch now supports OpenSSL 3 libraries for the TLS 1.3 compatibility that was missing from v5.18.0. The issue is resolved for the reported setup.
